packages/ra-ui-materialui/src/input/TranslatableInputs.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { ReactElement, ReactNode } from 'react';
+import {
+    TranslatableContextProvider,
+    useTranslatable,
+    UseTranslatableOptions,
+} from 'ra-core';
+import { TranslatableInputsTabs } from './TranslatableInputsTabs';
+import { TranslatableInputsTabContent } from './TranslatableInputsTabContent';
+
+/**
+ * Provides a way to edit multiple languages for any inputs passed as children.
+ *
```

Comment:
Could you add an example value format, as in the doc? It helps understanding when to use it.

Reply:
Done